napari/utils/colormaps/colormap_utils.py

```diff
@@ -374,11 +433,18 @@ def increment_unnamed_colormap(
     -------
     name : str
         Name of colormap after incrementing.
+    display_name : str
+        Display name of colormap after incrementing.
     """
+    display_name = trans._('[unnamed colormap]')
     if name == '[unnamed colormap]':
         past_names = [n for n in existing if n.startswith('[unnamed colormap')]
         name = f'[unnamed colormap {len(past_names)}]'
-    return name
+        display_name = trans._("[unnamed colormap {number}]").format(
+            number=len(past_names)
+        )
+
+    return name, display_name
```

Comment:
I changed this return type to a tuple, is this ok, or is this considered a breaking change?

is `increment_unnamed_colormap` expected to be used outisde napari?

Reply:
we could make method private `_increment_unnamed_colormap`, i don't expect it to be used outside napari, and am fine with the breaking change

Reply:
Sounds like a plan!

Reply:
Let's make it private, that makes it easy to create a deprecated compatibility version if someone complains at pre-release. @Czaki are you using this function?

Reply:
I do not use this function. I provide name for colormap. More problematic for me may be the need of providing pair of names (`name`, `display_name`) but comments suggest that it will be solved.